master/buildbot/process/buildstep.py

```diff
@@ -1229,7 +1231,7 @@ def regex_log_evaluator(cmd, _, regexes):
         # we won't be changing "worst" unless possible_status is worse than it,
         # so we don't even need to check the log if that's the case
         if worst_status(worst, possible_status) == possible_status:
-            if isinstance(err, (basestring)):
+            if isinstance(err, (string_types)):
```

Comment:
You are correct, the parenthesis is not needed here.  It was there before my change,
but it wasn't needed before.  I can take it out.  That is not a problem.